src/sys/context.rs

```diff
@@ -0,0 +1,25 @@
+use libc;
```

Comment:
I did not notice that, but should have thought about it. Thank you. This might be a good addition to our conventions file.